foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,54 @@
+1. Your game plays against the computer—create a function to `return` the computer's choice:
+   1. Create a new function called `getComputerChoice`.
+   1. Make `getComputerChoice` *randomly* `return` one of the following string values: `"Rock"`, `"Paper"` or `"Scissors"`.
+   1. Remember to test what your function returns with [browser's developer tools](https://www.theodinproject.com/lessons/foundations-javascript-developer-tools) or `console.log`.
+1. Your game plays turn by turn—create a function to take the human and computer choices, then `return` the round winner:
+   1. Create a new function called `playRound`.
+   1. Define two parameters on `playRound`: `playerSelection` and `computerSelection`. Use those two parameters to take the human and computer choices as arguments.
+   1. Make `playRound` `return` a string value representing the round winner, such as: `"You Lose! Paper beats Rock"`.
+
+   ~~~javascript
+   function playRound(playerSelection, computerSelection) {
+   // your code here!
+   }
  
-    const playerSelection = "rock";
-    const computerSelection = getComputerChoice();
-    console.log(playRound(playerSelection, computerSelection));
-    ~~~
-
-6.  Write a NEW function called `game()`. Use the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
-    *   You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
-    *   At this point you should be using `console.log()` to display the results of each round and the winner at the end.
-    *   Use `prompt()` to get input from the user. [Read the docs here if you need to.](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
-    *   Feel free to re-work your previous functions if you need to. Specifically, you might want to change the return value to something more useful.
-    *   Feel free to create more "helper" functions if you think it would be useful.
+   const playerSelection = "rock";
+   const computerSelection = getComputerChoice();
+   console.log(playRound(playerSelection, computerSelection));
+   ~~~
+
+1. Your game plays for 5 rounds, and reports the winner or loser after the 5 rounds—create a function to represent the game:
+   1. Create a new function called `game`.
+   1. Get the human player choice using `prompt` function. To learn about `prompt`, check the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
+   1. Put the previous function—`playRound`—inside `game`.
+   1. Play 5 rounds by calling `playRound` 5 times.
+   1. If you know about loops, use them. If not, don't worry! We cover loops in the next lesson. 
```

Comment:
```suggestion
   - Create a new function called `game`.
   - Get the human player choice using the `prompt` method. Read [this MDN article about prompts](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) if you need to learn more about it.  
   - Put the previous `playRound` function inside `game`.
   - Play 5 rounds by calling `playRound` 5 times.
   - Re-work your previous functions or create more helper functions if you feel the need to. Specifically, you might want to change the return values to something more useful.
   - If you already know about loops, you can use them. If not, don't worry! We cover loops in the next lesson. 
```
- descriptive/accessible link 
- important hint about the return values added back in (needed for score and winner tracking)